GOVERNANCE.md

```diff
@@ -0,0 +1,21 @@
+# Reference Types Working Group Governance
+
+## Discussion
+
+Discussions take place on GitHub's [discussion board](https://github.com/opencontainers/wg-reference-types/discussions) and during [weekly meetings](https://github.com/opencontainers/wg-reference-types/blob/main/README.md#meetings). Notes are taken down at the meetings are recorded in [this google document](https://docs.google.com/document/d/1SVOWQTowigXzbYdorzfa7tMmrcm91yK12LvSONqziJY/edit). Initial seeding of the GitHub discussion topics take place in [this google document](https://docs.google.com/document/d/1ky0LMAvWL_XRnfId9h5GBswV7GGko5GejuyaMWCGX-Y/edit#). It is expected that these google documents will be deprecated in favor of the GitHub Discussion board and GitHub issues.
+
+Discussions shall follow the [neutral framework](https://github.com/opencontainers/wg-reference-types/blob/main/docs/PERSONAS.md) previously decided by this working group. Changes to the framework are encouraged via GitHub discussions, issues, and pull requests.
+
+Working group members and maintainers shall follow the [OCI code of conduct](https://github.com/opencontainers/.github/blob/master/CODE_OF_CONDUCT.md)
+
+## Documentation
+
+Meeting minutes shall be published on this GitHub repository. A notetaker shall be identified to take notes during the meetings. Consensus shall be documented in the form of GitHub issues that can be referenced in pull requests.
+
+## Maintainer Operations
+
+Maintainers shall be responsible for overseeing documentation of community decisions, consensus, and resolution of conflicts. Maintainers shall provide timely feedback to requests for review and promptly merge approved Pull Requests. Maintainers shall operate in good faith and trust in the community and in each other. Maintainers shall keep the documentation, issues, and discussion topics as up-to-date as possible. The Service Level Agreement for responses is no more than 5 business days. In the event a Maintainer cannot perform their duties, an interim or permanent replacement shall be nominated from the community. Any member from the community may volunteer to be a maintainer as long as they pledge to follow the OCI code of conduct and perform these duties. 
+
+## Conflict Resolution
+
+In the event of a conflict, group members are encouraged to fall back on the [neutral framework for discussion](https://github.com/opencontainers/wg-reference-types/blob/main/docs/PERSONAS.md). Conflicts that cannot be resolved within the group shall be referred to the OCI Technical Oversight Board (TOB) as per [section 5.xv](https://github.com/opencontainers/tob/blob/main/CHARTER.md#5-technical-developer-community) of the TOB Charter. Resolution of the conflict shall be documented on this GitHub respository's issues which can be referred to in a pull request.
```

Comment:
same suggestion as @lachie83 made above re: "neutral framework" => "documented personas"

Reply:
Oops! Forgot this one